static/app/views/issueDetails/streamline/eventNavigation.tsx

```diff
@@ -153,16 +175,21 @@ export const EventNavigation = forwardRef<HTMLDivElement, EventNavigationProps>(
     return (
       <div {...props} ref={ref}>
         <EventNavigationWrapper>
-          <Tabs>
+          <Tabs value={selectedOption}>
             <TabList hideBorder variant="floating">
               {EventNavOrder.map(label => {
+                const eventPath =
+                  label === selectedOption
+                    ? undefined
+                    : {
+                        pathname: normalizeUrl(baseEventsPath + label + '/'),
+                        query: {...location.query, referrer: `${label}-event`},
+                      };
                 return (
                   <TabList.Item
-                    to={{
-                      pathname: normalizeUrl(baseEventsPath + label + '/'),
-                      query: {...location.query, referrer: `${label}-event`},
-                    }}
+                    to={eventPath}
                     key={label}
+                    hidden={label === 'custom' && selectedOption !== 'custom'}
```

Comment:
nit: Just so they don't drift down the line, you can replace all these one off strings of `'custom'` with `EventNavOptions.CUSTOM.`

Reply:
good idea!